src/raftstore/store/snap.rs

```diff
@@ -165,6 +179,9 @@ impl SnapFile {
         if let Some((mut f, path)) = self.tmp_file.take() {
             try!(f.write_u32::<BigEndian>(self.digest.sum32()));
             try!(f.flush());
+            let file_len = try!(f.seek(SeekFrom::End(0)));
+            let mut size_track = self.size_track.wl();
+            *size_track = size_track.saturating_add(file_len);
```

Comment:
em, I don't think we need saturating here, u64 overflow seems impossible, expect bug.